src/index.ts

```diff
@@ -1413,6 +1413,45 @@ export class MintSDK {
     })) as ethers.providers.TransactionResponse
   }
 
+  /**
+   * Returns whether the contract has specified nft
```

Comment:
I thought the text was missing.

Return whether the wallet has specified an nft.